lib/connection.js

```diff
@@ -1101,14 +1101,16 @@ Connection.prototype.loginBySoap = function(username, password, callback) {
 /**
  * Logout the current session
  *
+ * Setting revoke to true will revoke API access
+ *
  * @param {Callback.<undefined>} [callback] - Callback function
  * @returns {Promise.<undefined>}
  */
-Connection.prototype.logout = function(callback) {
+Connection.prototype.logout = function(callback, revoke = false) {
```

Comment:
The argument order should set callback to last.

Reply:
Updated in 9699c90